modules/gatsby/src/components/styled/typography.js

```diff
@@ -0,0 +1,67 @@
+import {styled} from 'baseui';
+
+// h4, h5, and h6 haven't been spec'ed yet, so we'll just reuse baseweb defaults
+export {H4, H5, H6} from 'baseui/typography';
+
+const monospaceFontStack = `'SF Mono', Consolas, Menlo, Monaco, 'Andale Mono WT', 'Andale Mono', 'Lucida Console', 'Lucida Sans Typewriter', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', 'Liberation Mono', 'Nimbus Mono L', 'Courier New', Courier, monospace`;
```

Comment:
What does this do? Various fallbacks for the fixe width font? Worth a comment for those of us who are not baseui/styling fiends?